jupyter_server/terminal/api_handlers.py

```diff
@@ -1,16 +1,22 @@
 import json
 from tornado import web
 from ..base.handlers import APIHandler
+from jupyter_server.services.auth.decorator import authorized
+
+
+RESOURCE_NAME = "terminal"
```

Comment:
Most resources are plural and it seems like this would be similar to those (e.g., files, contents, kernels, kernelspecs).